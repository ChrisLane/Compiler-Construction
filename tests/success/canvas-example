// The same code as found on Canvas except for the semicolon at the end. This should parse.
main() {int x=1; x=x+1;return x;}
