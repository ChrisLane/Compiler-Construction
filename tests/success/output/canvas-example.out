main (  ) { 
    New ( "x" = Const 1 ) In { 
        Asg ( "x" := Operator ( Plus, "x" , Const 1 ) ) ; 
        Deref ( "x" ) 
    } 
}
